babel/src/main/codegen/config.fmpp

```diff
@@ -922,6 +924,11 @@ data: {
       "InlineModOperator()"
     ]
 
+    dateTimeExpressionSuffixes: [
+      "SqlDateTimeAtLocal"
```

Comment:
Suggest you remove the existing entries from this list, insert DateTimeTerm, and then use the <#list> syntax in Parser.jj to iterate through all the functions in this list (which will only have one at this point in time).

Reply:
Change is made https://github.com/googleinterns/calcite/pull/51/commits/74cf85b23c39b842cf19b77b2430eabfe901c1b4